### ISSUE TITLE
Categories with unicode character in their name malfunctions

### ISSUE THREAD

**Issue by leonard-IMBERT** [xvw/muhokama#32]
Right now categories can have in their name unicode characters such as 🬃 (https://unicode-table.com/fr/1FB03/).

Because categories are used in the URI in `/topic/by/category/:cat`, when trying to access the category the generated URI is `localhost:4000/topic/by/category/🬃`. When accessing the category it show no topics (even if there are ones)

I suggest that we limit the categories name following the [URI rfc](https://datatracker.ietf.org/doc/html/rfc3986) for simplicity.

**Comment by gr-im** [comment]
Yes, it was mentionned here: https://github.com/xvw/muhokama/issues/23
Since ATM, the owner of categories is @xvw, it is not in "high priority" but I'll keep that in my mind!
